frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -103,18 +137,17 @@ const EditableRow: React.FC<EditableRowProps> = ({
   );
 };
 
-interface MetadataTableProps {
+export interface MetadataTableProps {
   data: RowData[];
   onUpdate?: (id: string, value: unknown) => void;
   variant?: Size;
```

Comment:
is there a use case for this prop? i didn't see it as part of any of the existing designs

Reply:
I wasn't sure if we should keep it or not. My main thought was if we wanted to embed this within a full page dashboard and make it smaller for informational purposes. I can remove for now though.